test/abstract_syntax_tree.coffee

```diff
@@ -3731,8 +3709,6 @@ test "AST as expected for Switch node", ->
       ]
     ]
 
-#   # TODO: File issue for compile error when using `then` or `;` where `\n` is rn.
```

Comment:
I'm not sure what this was referring to

Reply:
Probably `then\r\n` as opposed to the typical `then\n`.

Reply:
I forgot that there. It's covered in #5127.